sysfs/vulnerability.go

```diff
@@ -59,29 +73,29 @@ func (fs FS) CPUVulnerabilities() ([]Vulnerability, error) {
 // Vulnerability represents a single vulnerability extracted from /sys/devices/system/cpu/vulnerabilities/.
 type Vulnerability struct {
 	CodeName   string
-	State      string
+	State      int
 	Mitigation string
 }
 
-func parseVulnerability(name, value string) (Vulnerability, error) {
-	v := Vulnerability{CodeName: name}
-	value = strings.TrimSpace(value)
-	if value == notAffected {
-		v.State = notAffected
+func parseVulnerability(name, rawContent string) (*Vulnerability, error) {
+	v := &Vulnerability{CodeName: name}
+	rawContent = strings.TrimSpace(rawContent)
+	if rawContent == notAffected {
```

Comment:
I wonder if we should `strings.ToLower(rawContent)` to make this less fragile.

Reply:
that's a good point

I've pushed another version which unifies character cases when parsing the state, let me know what you think.

I've also added test cases for vulnerable state